ShippingZones.tsx

```diff
@@ -50,6 +50,17 @@ const ShippingZonesList = () => {
 
   const navigation = useNavigation();
 
+  React.useEffect(() => {
```

Comment:
[Q] Did you choose to use a different `useEffect` hook for some reason? I'm not sure what is the "correct" way 😅

Reply:
The same reason as [here](https://github.com/woocommerce/WooCommerce-Shared/pull/37#discussion_r1242588412). I've removed prefix here as well, thanks for noticing!

Reply:
Oh, I meant here about having two `useEffect` functions on the file 🤔